spec/fare_transactions.schema.json

```diff
@@ -8,151 +8,214 @@
     {
       "name": "date",
       "type": "date",
-      "required": "required",
-      "description": "Service date.",
-      "external_reference": "GTFS"
+      "description": "Service date. References GTFS",
+      "constraints": {
+        "required": true
+      }
     },
     {
       "name": "timestamp",
-      "type": "POSIX time",
-      "required": "required",
+      "type": "time",
```

Comment:
Since the service date might not equal the event date this should be type "timestamp". For example, date = 2022-08-21, timestamp 2022-08-22T06:10:08Z for service that operated early Monday morning as part of the Sunday schedule. The frictionless "time" field doesn't seem to support time beyond the standard 24 hours.

Reply:
Deferring resolution per #31